contrib/k8s-domain-exporter.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config
```

Comment:
I'd name this `domain-exporter` instead of "config" to not overlap with another ConfigMap.